tests/nexus/test_nexus.py

```diff
@@ -19,7 +19,6 @@
 
 import logging
 import os
-import difflib
```

Comment:
Why did you remove this line? It breaks the test for me...

Reply:
Sorry, that was not intentionally removed, but I see that somehow erased. 
Thanks @lukaspie.